src/components/network/layers/arrow-layer-vertex.glsl.js

```diff
@@ -128,6 +128,19 @@ mat3 calculateRotation(vec3 commonPosition1, vec3 commonPosition2) {
               0,           0,           0);
 }
 
+// Adjustment factor for low zoom levels
```

Comment:
Did you write this code ? Or add a comment indicating the source ?

Reply:
@jonenst It comes from the deckgl library (https://github.com/visgl/deck.gl/blob/401d624c0529faaa62125714c376b3ba3b8f379f/modules/core/src/shaderlib/project/project.glsl.ts#L65) but we don't have access to these methods. The idea here is only to override the use of project.geometry.y for the projection by the actual latitude.
I update the code.